testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -690,9 +690,63 @@ impl PreviouslyVerifiedTestData {
         user_id!("@carol:localhost")
     }
 
-    /// Current user keys query response containing the cross-signing keys
+    pub fn own_unsigned_device_id() -> &'static DeviceId {
+        device_id!("AHIVRZICJK")
+    }
+    pub fn own_signed_device_id() -> &'static DeviceId {
+        device_id!("LCNRWQAVWK")
+    }
```

Comment:
Could you add some doc-comments to these, explaining what they are, and linking to `own_keys_query_response_1`?

Reply:
I added some more comments https://github.com/matrix-org/matrix-rust-sdk/pull/3810/commits/3fa6891ad8125254d600381000ba914c6136a548

There is probably a better way to organize these keys/query response. Would need to put some more thinking on it. But basically there could be a full template response, and one could just add/remove devices as needed, add/remove signatures as needed.
That would allow for more flexible test configuration.

Reply:
> There is probably a better way to organize these keys/query response.

Yes, I don't love this system. But we don't need to solve that right now.